README.md

```diff
@@ -13,6 +13,7 @@ Access design documentation for features slotted in the 4.2 release of OpenShift
 - [Cron Jobs](http://openshift.github.io/openshift-origin-design/web-console/future-openshift/cron-jobs/cron-jobs)
 - [Dashboards and Projects](http://openshift.github.io/openshift-origin-design/web-console/future-openshift/dashboards/dashboards)
 - [Diagnostics (WIP)](https://docs.google.com/document/d/11rQliTE7fdyqDWFc5F-w-jCBRxxNBymngidQ6xlCiws/edit?usp=sharing)
+- [Health Checks](http://openshift.github.io/openshift-origin-design/web-console/future-openshift/health-checks/health-checks)
```

Comment:
Should we put this in the 4.3 section?

Reply:
Ah, yes probably :)

Reply:
is this definitely in 4.3 @bmignano 🙏 ?

Reply:
I don't know about definitely, but fingers crossed! 🤞